sdks/python/apache_beam/io/bigquery.py

```diff
@@ -862,11 +925,18 @@ def convert_row_to_dict(self, row, schema):
       elif field.type == 'TIME':
         # Input: "00:49:36" --> Output: "00:49:36"
         value = value
-      else:
+      elif field.type == 'DATE':
```

Comment:
Use 'or' for DATE, TIME, and DATETIME fields and add a comment saying we don't do any special handling of these types.